core/dbt/cli/params.py

```diff
@@ -248,6 +269,13 @@
     type=click.Path(file_okay=True, dir_okay=False, writable=True),
     default=None,
 )
+package = click.option(
+    "--package", envvar=None, help="Name of package to add to packages.yml.", type=click.STRING
+)
+
+package_version = click.option(
+    "--package-version", envvar=None, help="Version of the package to install.", type=click.STRING
+)
```

Comment:
have we considered removing these two options in favor with providing a `package@version` (or `package=version` or just `package`) as the option value for `--add`? I think that would be more intuitive from a user perspective (fewer flags to reason over), and could simplify option validation. It would just require a bit of parsing internally to get the package and package_version out of the provided `--add` option value.

Reply:
Used @ in the end